pkg/volume/flexvolume/plugin.go

```diff
@@ -34,8 +34,9 @@ import (
 )
 
 const (
-	flexVolumePluginName       = "kubernetes.io/flexvolume"
-	flexVolumePluginNamePrefix = "flexvolume-"
+	flexVolumePluginName = "kubernetes.io/flexvolume"
+	// for historical reasons, keep blank to be compatible with past versions
+	flexVolumePluginNamePrefix = ""
```

Comment:
shall we just simply remove this prefix? it's not exported and not included in volume path anyway.